gateway/dataimpl/server_v1/queryadminserver.go

```diff
@@ -395,6 +618,11 @@ func (s *QueryIndexAdminServer) BuildDeferredIndexes(
 	ctx context.Context,
 	in *admin_query_v1.BuildDeferredIndexesRequest,
 ) (*admin_query_v1.BuildDeferredIndexesResponse, error) {
+	err := s.validateNames(nil, &in.BucketName, in.ScopeName, in.CollectionName)
```

Comment:
Given the current behaviour, I thought this might be a good temporary measure to catch typos and invalid names since the get all will always successfully execute